index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Nemwel Boniface">
+    <meta name="description" content="This is my personal portfolio site that I will share to all potential employers to showcase my skills as a developer">
+
+    <!--google Fonts roboto, crete-round-->
+    
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+
+    <!-- font awesome icons -->
+
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <title>Nemwel Portfolio</title>
+
+    <!--Page stylesheets-->
+    <link rel="stylesheet" href="stylesheets/landing.css">
+
+    <!--Smooth scroll script-->
+       
+    <script src="https://cdn.jsdelivr.net/gh/cferdinandi/smooth-scroll/dist/smooth-scroll.polyfills.min.js"></script>
+
+</head>
+<body>
```

Comment:
- [ ] I can see an empty space by the side of your page. It seems the illustrator added extra space to the side of your page. Please try to position the illustrator properly to fix this 💪 

<img width="203" alt="Loom_FR8DJA91Ve" src="https://user-images.githubusercontent.com/81468675/148491288-1a143a68-fbb7-4b42-8eeb-d75e1f6ff50a.png">

Reply:
Hello @julie-ify Thank you for this review.
- I have tried to have a look at what might be causing my illustrator to overflow to the right and I have not been able to make the change.
- Could you kindly recommend something that I should do to fix the overflow?

Reply:
Hey @julie-ify  I managed to sort it out!